tests/google.js

```diff
@@ -5,33 +5,34 @@ const sleep = 3000;
 
 async function runTest() {
   // abre o navegador
-  const driver = await new Builder().forBrowser(Browser.CHROME).build();
+  const driver = await new Builder().forBrowser(Browser.CHROME).build()
 
   // maximiza a janela do browser
-  await driver.manage().window().maximize();
+  await driver.manage().window().maximize()
 
-  await driver.sleep(sleep);
+  await driver.sleep(sleep)
 
   // acessa o google
-  await driver.get('https://google.com');
+  await driver.get("https://google.com");
 
   await driver.sleep(sleep);
 
   // pesquisa por 'programming memes'
-  await driver.findElement(By.name('q')).sendKeys('programming memes', Key.RETURN);
+  await driver.findElement(By.name('q')).sendKeys("programming memes", Key.RETURN);
```

Comment:
Codacy has a fix for the issue: [Strings must use singlequote.](https://app.codacy.com/gh/mauricio-cantu/selenium-presentation/pullRequest?prid=9606874)
```suggestion
  await driver.findElement(By.name('q')).sendKeys('programming memes', Key.RETURN);
```